public/app/features/explore/Explore.tsx

```diff
@@ -71,36 +60,12 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
 });
 
 export interface ExploreProps {
-  changeSize: typeof changeSize;
-  datasourceInstance: DataSourceApi | null;
-  datasourceMissing: boolean;
   exploreId: ExploreId;
-  modifyQueries: typeof modifyQueries;
   scanning?: boolean;
   scanRange?: RawTimeRange;
-  scanStart: typeof scanStart;
-  scanStopAction: typeof scanStopAction;
-  setQueries: typeof setQueries;
-  queryKeys: string[];
-  isLive: boolean;
-  syncedTimes: boolean;
-  updateTimeRange: typeof updateTimeRange;
-  graphResult: DataFrame[] | null;
-  logsResult?: LogsModel;
-  absoluteRange: AbsoluteTimeRange;
-  timeZone: TimeZone;
   onHiddenSeriesChanged?: (hiddenSeries: string[]) => void;
-  queryResponse: PanelData;
   originPanelId: number;
```

Comment:
I think `originPanelId, onHiddenSeriesChanged, scanRange, scanning` are still can be removed.

Reply:
Yeah you're right, removed them now 👍